### DISCUSSION TITLE
Gemini 2.0 api access

### DISCUSSION THREAD

**Discussion by mseigal** [danielmiessler/Fabric#1196] [General]
Just had a look at the Gemini models and see there is not yet access to Gemini 2.0 models. Is this a Google imposed limitation or will Fabric need to be updated to include the newer models?

**Comment by marcelmatula** [comment]
I see it under name "gemini-2.0-flash-exp". I do not have paid version. Just created api key via Google AI Studio.

./fabric -L

![image](https://github.com/user-attachments/assets/ca9f531c-1f75-4d98-b86d-8f534be8b11f)
